flight/Modules/Sensors/sensors.c

```diff
@@ -357,6 +361,10 @@ static void update_accels(struct pios_sensor_accel_data *accels)
 	    accels->z * accel_scale[2] - accel_bias[2]
 	};
 
+	accels_out[0] = filter_execute(&accel_filter, 0, accels_out[0]);
```

Comment:
Style choice-- it would be possible to just pass in accels_out and do all 3 at once.  Not sure which is better-- this is perhaps a little more versatile; the other is a bit quicker.